src/main/secret/secret_manager.cpp

```diff
@@ -283,8 +282,8 @@ BoundStatement SecretManager::BindCreateSecret(CatalogTransaction transaction, C
 	}
 
 	BoundStatement result;
-	result.names = {"Success"};
-	result.types = {LogicalType::BOOLEAN};
+	result.names = {"Success", "Secret"};
+	result.types = {LogicalType::BOOLEAN, LogicalType::MAP(LogicalType::VARCHAR, LogicalType::VARCHAR)};
```

Comment:
We are still setting the `StatementReturnType` of `CREATE SECRET` to `StatementReturnType::NOTHING` in `bind_create.cpp` - meaning no actual result set will be printed/shown to the user in most scenarios. Should we change that as well?